theme/templates/resource-landing-page/modals.html

```diff
@@ -1395,412 +1395,497 @@ <h4 class="modal-title">Manage access</h4>
-                                                    <a class="btn-change-share-permission" data-arg="share-edit-{{ u.pk }}">Can edit</a>
-                                                </form>
-                                            </li>
-                                            <li data-access-type="Is owner" {% if not is_owner_user %}class="disabled"{% endif %}>
-                                                <form id="share-owner-{{ u.pk }}" class="share-form-owner" data-access-type="Is owner"
-                                                      action="/hsapi/_internal/{{ cm.short_id }}/share-resource-with-user/owner/{{ u.pk }}/"
-                                                      method="POST">
-                                                    {% csrf_token %}
-                                                    <a class="btn-change-share-permission" data-arg="share-owner-{{ u.pk }}">Is owner</a>
-                                                </form>
-                                            </li>
-                                        </ul>
-                                    </span>
-                                </td>
-                                <td class="user-actions">
-                                    <form id="form-remove-user-{{ u.pk }}" class="remove-user-form"
-                                          action="/hsapi/_internal/{{ cm.short_id }}/unshare-resource-with-user/{{ u.pk }}/"
-                                          method="POST">
-                                        {% csrf_token %}
-                                        <span class="btn-unshare-resource glyphicon glyphicon-remove btn-remove-row"></span>
-                                    </form>
-                                </td>
-                            </tr>
-                            {% endfor %}
-
-                            {# Groups who can edit the content of this resource #}
-                            {% for u in edit_groups %}
-                            <tr id="row-id-{{ u.pk }}">
-                                <td>
-                                    <div class="user-scope">
-                                        <div class="group-image-wrapper extra-small">
-                                            {% if u.gaccess.picture and u.gaccess.picture.url %}
-                                                <div class="group-image-extra-small group-preview-image-default"
-                                                 style="background-image: url({{ u.gaccess.picture.url }})"></div>
+                                    </td>
+                                </tr>
+
+                                {# Owners with full permissions to do anything on this resource #}
+                                {% for u in owners %}
+                                <tr id="row-id-{{ u.pk }}" {% if owners|length == 1 or not is_owner_user %}class="hide-actions"{% endif %}>
```

Comment:
@hyi, this logic disables changing permissions for owner items if there is only one owner or if the current user is not an owner. This is working as intended.